tasks/main.yml

```diff
@@ -8,17 +8,19 @@
   file:
     path: "{{letsencrypt_directory}}"
     state: directory
+    mode: 0777
+  become: yes
 
 - name: download certbot-auto
   get_url: 
     url: https://dl.eff.org/certbot-auto
-    dest: ~/
+    dest: "{{letsencrypt_directory}}"
     mode: 0755
   become: yes
   become_user: "{{letsencrypt_user}}"
 
 - name: generate cert
-  shell: ./certbot-auto -n certonly --server https://acme-v01.api.letsencrypt.org/directory -a webroot --webroot-path={{letsencrypt_directory}} --agree-tos --email {{letsencrypt_email}} {{letsencrypt_domainstring}}
+  shell: "{{letsencrypt_directory}}/./certbot-auto -n certonly --server https://acme-v01.api.letsencrypt.org/directory -a webroot --webroot-path={{letsencrypt_directory}} --agree-tos --email {{letsencrypt_email}} {{letsencrypt_domainstring}}"
```

Comment:
Does this work well? I would think yoy should do something like `cd {{letsencrypt_directory}} && ./certbot etcetc`, but if this actually works fine, then I don't have a problem with it!